modules/gltf/gltf_document.cpp

```diff
@@ -4978,40 +4991,15 @@ GLTFMeshIndex GLTFDocument::_convert_mesh_instance(Ref<GLTFState> state, MeshIns
 	if (p_mesh_instance->get_mesh().is_null()) {
 		return -1;
 	}
-	Ref<EditorSceneImporterMesh> import_mesh;
-	import_mesh.instantiate();
-	Ref<Mesh> godot_mesh = p_mesh_instance->get_mesh();
-	if (godot_mesh.is_null()) {
+	Ref<ArrayMesh> import_mesh = p_mesh_instance->get_mesh();
```

Comment:
Not all meshes are ArrayMesh. If I export a primitive BoxMesh, it should work

Reply:
Changed to Ref<Mesh>